docs/src/modules/java/examples/testkit/java-eventsourced-shopping-cart/intarget/generated-test-sources/akkaserverless/com/example/shoppingcart/domain/ShoppingCartTestKit.java

```diff
@@ -0,0 +1,23 @@
+package com.example.shoppingcart.domain;
+
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntity;
+import com.akkaserverless.javasdk.testkit.Result;
+import com.akkaserverless.javasdk.testkit.impl.AkkaServerlessTestKitHelper;
+import com.example.shoppingcart.ShoppingCartApi;
+import com.google.protobuf.Empty;
+import java.util.List;
+
+public class ShoppingCartTestKit {
+
+
+  public ShoppingCartTestKit(ShoppingCart entity) // <1>
+
+  public ShoppingCartTestKit(ShoppingCart entity, ShoppingCartDomain.Cart state) // <2>
```

Comment:
This is different now with a private constructor and `.of` factories, less important but the imports are also a bit different.